BedrockServer.cpp

```diff
@@ -825,18 +826,6 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
     canWriteParallel = canWriteParallel && (state == SQLiteNode::LEADING);
     canWriteParallel = canWriteParallel && (command->writeConsistency == SQLiteNode::ASYNC);
 
-    // If all the other checks have passed, and we haven't sent a quorum command to the sync thread in a while,
```

Comment:
This block has moved to where we actually need to write. This is to avoid sending slow read commands to the sync thread for quorum, which is useless anyway.